src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,48 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+import { Tab } from '../../types/Tab';
+
+type Props = {
+  tabs: Tab[];
+  selectedTabId: string;
+  handleSelect: (tab: Tab) => void;
+};
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  handleSelect,
+}) => {
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => {
+            const { id, title } = tab;
+
+            return (
+              <li
+                className={classNames(
+                  selectedTabId === id && 'is-active',
+                )}
+                data-cy="Tab"
+              >
+                <a
+                  href={`#${id}`}
+                  data-cy="TabLink"
+                  onClick={() => handleSelect(tab)}
+                >
+                  {title}
+                </a>
+              </li>
+            );
+          })}
+        </ul>
+      </div>
+      <div
+        className="block"
+        data-cy="TabContent"
+      >
+        {tabs.find(tab => selectedTabId === tab.id)?.content}
+      </div>
```

Comment:
This block would be better move in App, because "selectedTab" is already there and you can simply write "selectedTab.content" . Now you are looking for "selectedTab" in App , and then you are looking for it in "Tabs"

Reply:
<img width="795" alt="Screenshot 2022-09-26 at 23 04 29" src="https://user-images.githubusercontent.com/47606379/192369462-06833866-be60-4cff-886f-56e945ddb2b6.png">